src/hoodie/request.js

```diff
@@ -145,6 +152,14 @@ exports.HTTP_STATUS_ERROR_MAP = {
   500: 'HoodieServerError'
 };
 
+
+/**
+ * pouch-ajax already parses the error and doesn't pass
+ * along the xhr instance for us to be able to parse the
+ * error here ourselves
+ * so I'm not quite sure how to port this functionality over
+ * without losing some of it
+ */
 exports.parseErrorFromResponse = function(xhr) {
   var error = JSON.parse(xhr.responseText);
```

Comment:
As above. `xhr` isn't exposed, and `pouchdb-ajax` already formats the error, so this function is redundant right now, I've just left it in for discussion.